### PR TITLE
Enhancement: update account page ui design

### PR DESCRIPTION
**Pull request by hasinur1997**

### CODE REVIEW

**Comment by FaraziF** [comment]
In design requirements, in the post title column image option missing. 
Design Phase: [image](https://prnt.sc/10m4jn3)
Development Phase: [image](https://prnt.sc/110gm26)

**Comment by zamandevs** [comment]
@FaraziF FYI there is an option for showing feature image on our global dashboard settings 'show featured image'. 
Everything is fine i see. if you please check again.

File: assets/less/frontend-forms.less [hunk 1]
```diff
@@ -3,8 +3,10 @@
 @mediaXS: 767px;
 @mediaSM: 991px;
 @mediaMD: 1200px;
-
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;1,300&display=swap');
```
Comment (tareq1988) [inline assets/less/frontend-forms.less:6]: We should not import the font as it may conflict with the theme.

File: templates/dashboard/posts.php [hunk 2]
```diff
@@ -122,25 +118,28 @@
                                 ?>
                             </td>
                         <?php } ?>
-                        <td>
-                            <?php if ( !$show_link ) { ?>
+                        <td data-label="<?php _e( 'Title: ', 'wp-user-frontend' ); ?>">
```
Comment (tareq1988) [inline templates/dashboard/posts.php:121]: In attributes, we should use `esc_attr_e()`.

File: templates/dashboard/posts.php [hunk 3]
```diff
@@ -122,25 +118,28 @@
...
-                                <a href="<?php the_permalink(); ?>" title="<?php printf( esc_attr__( 'Permalink to %s', 'wp-user-frontend' ), the_title_attribute( 'echo=0' ) ); ?>" rel="bookmark"><?php the_title(); ?></a>
+                                <a href="<?php the_permalink(); ?>" title="<?php printf( esc_attr__( 'Permalink to %s', 'wp-user-frontend' ), the_title_attribute( 'echo=0' ) ); ?>" rel="bookmark"><?php echo wp_trim_words( get_the_title(), 5 ); ?></a>
 
                             <?php } ?>
+                            <span class="post-edit-icon">
+                                &#x25BE;
+                            </span>
                         </td>
-                        <td>
+                        <td data-label="<?php _e( 'Status: ', 'wp-user-frontend' ); ?>" class="data-column">
```
Comment (tareq1988) [inline templates/dashboard/posts.php:135]: Use `esc_attr_e`

File: templates/dashboard/posts.php [hunk 4]
```diff
@@ -122,25 +118,28 @@
...
                             <?php wpuf_show_post_status( $post->post_status ); ?>
                         </td>
 
                         <?php do_action( 'wpuf_account_posts_row_col', $args, $post ); ?>
 
                         <?php if ( 'on' == $enable_payment && 'off' != $payment_column ) { ?>
-                            <td>
+                            <td data-label="<?php _e( 'Payment: ', 'wp-user-frontend' ); ?>" class="data-column">
```
Comment (tareq1988) [inline templates/dashboard/posts.php:142]: Use `esc_attr_e`

File: templates/dashboard/posts.php [hunk 5]
```diff
@@ -151,7 +150,7 @@
                                     </td>
                                 <?php } ?>
 
-                                <td>
+                                <td data-label="<?php _e( 'Options: ', 'wp-user-frontend' ); ?>" class="data-column">
```
Comment (tareq1988) [inline templates/dashboard/posts.php:153]: Use `esc_attr_e`

**Review by tareq1988 — CHANGES_REQUESTED** [review]

File: assets/less/frontend-forms.less (hunk 1 above)
Comment (zamandevs) [reply assets/less/frontend-forms.less:6]: @tareq1988 If you please guide here for alternative implementation.

Comment (tareq1988) [reply assets/less/frontend-forms.less:6]: Just remove the font from here

**Pull request closed without merging**